app/views/mainstream_browse_pages/show.html.erb

```diff
@@ -2,6 +2,11 @@
   <%= link_to "Edit page",
     edit_mainstream_browse_page_path(@browse_page) %>
 
+  <% if @browse_page.has_children? %>
```

Comment:
We probably want this to be `@browse_page.can_have_children?`.  We might want to be able to set the `child_ordering` field, even when no children have yet been created.